packages/perseus-editor/src/__tests__/editor.test.tsx

```diff
@@ -80,8 +80,9 @@ describe("Editor", () => {
         render(<Harnessed onChange={onChangeMock} />);
 
         // Act
-        screen.getByRole("button", {name: "Remove image widget"}),
-            await userEvent.click();
+        await userEvent.click(
+            screen.getByRole("button", {name: "Remove image widget"}),
+        );
```

Comment:
question, nb: Did the syntax that was previously here not work? Or is this change mostly for readability?

Reply:
It was incorrect. It looks like deep in the library there's some defaulting to use the previous target as the target when `click()` is called, but the API types definitely don't allow for a missing element. I think it was sheer luck that this worked.